frontend/src/components/MapTooltip/MapTooltip.tsx

```diff
@@ -1,7 +1,11 @@
 import { LngLat } from 'mapbox-gl'
+import { IconButton } from 'material-ui'
```

Comment:
MaterialUI is packaged in a really annoying way where you have to be careful to import just the dependency you need, otherwise the whole thing gets pulled into bundle.js and adds a few MB of extra weight.

```diff
-import { IconButton } from 'material-ui'
+import { IconButton } from 'material-ui/IconButton'
```